release/models/platform/openconfig-platform-port.yang

```diff
@@ -129,7 +136,10 @@ module openconfig-platform-port {
         to the interfaces in this breakout group. This leaf need
         not be set if there is only one breakout group where all
         the interfaces are of equal speed and have equal number
-        of physical channels";
+        of physical channels.
+
+        The physical channels referred to by this leaf electrical channels
+        towards the transceiver.";
```

Comment:
minor editorial but should it say "...by this leaf are electrical..."?

Reply:
Fixed, thanks.